src/components/aboutus/index.jsx

```
@@ -68,30 +68,30 @@ function AboutUs() {
-                                    <Typography variant="paragraph" sx={{ width: { xs: "100%", md: "90%" }, fontFamily: "Roboto", lineHeight: { xs: "30px", md: "40px" } }} fontWeight="normal" fontSize={{ xs: "15px", md: "26px" }} color="#6E6B7B">
+                                <Stack direction="column" spacing={2} pt={1}>
+                                    <Typography variant="paragraph" sx={{ width: { xs: "100%", md: "82%" }, fontFamily: "Roboto", lineHeight: { xs: "30px", md: "35px" } }} fontWeight="normal" fontSize={{ xs: "15px", md: "18px" }} color="#6E6B7B">
                                         For decades, e-signatures have transformed the way businesses operate globally. As the market has grown, so too has the need for a more accessible and affordable solution.
                                     </Typography>
 
-                                    <Typography variant="paragraph" sx={{ alignSelf: "start", fontFamily: "Roboto", lineHeight: "23px" }} fontWeight="normal" fontSize={{ xs: "15px", md: "26px" }} color="#6E6B7B">
+                                    <Typography variant="paragraph" sx={{ alignSelf: "start", fontFamily: "Roboto", lineHeight: "23px" }} fontWeight="normal" fontSize={{ xs: "15px", md: "18px" }} color="#6E6B7B">
                                         That's where RequireSign comes in.
                                     </Typography>
 
                                     <a href={`${try_for_free}`} target='_blank' style={{ textDecoration: "none" }}>
```

Biome

[error] Avoid using target="_blank" without rel="noreferrer".

Opening external links in new tabs without rel="noreferrer" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noreferrer" attribute.


(lint/a11y/noBlankTarget)